### DISCUSSION TITLE
Make line indicator for installed add-ons in the Supervisor thicker

### DISCUSSION THREAD

**Discussion by JuanMTech** [home-assistant/frontend#11227] [Other feature requests]
## The request

For the add-ons in the Supervisor, the line that represents the installed add-ons is a little too thin. I would suggest making it a little thicker **(Example 1)**. Another option would be to have the line on the left **(Example 2)**.

### Example 1
![](https://i.imgur.com/3Rpcbjq.png)

### Example 2
![](https://i.imgur.com/k5ag9IP.png)

**Comment by spacegaier** [comment]
I personally vote for option 2. The horizontal line always looks confusing to me, as if it separates the entries in the columns into groups.
